robottelo/hosts.py

```diff
@@ -1084,6 +1073,17 @@ def configure_rhai_client(
         # Ensure insights-client rpm is installed
         if self.execute('yum install -y insights-client').status != 0:
```

Comment:
Can this be moved to below the registration to check if insights-client package is installed? 
and it can be installed with yum when `register=False`>

Reply:
In this case, I think logicially it should stay where it is... We need to make sure those packages are installed before we register

Reply:
@ColeHiggins2 yes, but this step would be covered with `setup_insights=register_insights` when `register=True`, where this isn't required, but its required to install insights-client manually, when `register=False`, right?